packages/vue/src/components/range/RangeSlider.jsx

```diff
@@ -200,11 +198,8 @@ const RangeSlider = {
 								height={4}
 								enable-cross={false}
 								{...{ props: this.$props.sliderOptions }}
-								{...{
-									on: {
-										input: this.handleSlider,
-									},
-								}}
+								onDrag-end={this.handleSlider}
+								tooltip="always"
```

Comment:
@mohdashraf010897 Move it to above line no 200 `sliderOptions` prop can have priority.

Reply:
fixed here https://github.com/appbaseio/reactivesearch/pull/1853/commits/6a1e2a831f23dbf88a79eed5f676751d1909daea